src/openApiClientJs/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "joanie-openapi-client-ts",
+  "version": "1.0",
```

Let's align version with mail and python packages

```suggestion
  "version": "1.1.0",
```